pkgs/development/tools/parsing/antlr/2.7.7.nix

```diff
@@ -10,6 +10,8 @@ stdenv.mkDerivation rec {
   patches = [ ./2.7.7-fixes.patch ];
   buildInputs = [ jdk ];
 
+  CXXFLAGS = lib.optionalString stdenv.isDarwin "-D_LIBCPP_ENABLE_CXX17_REMOVED_UNARY_BINARY_FUNCTION";
```

Comment:
I'm guessing this condition should actually be `stdenv.cc.libcxx != null` - this should fix it should anyone want to build it using clang/libcxx on a non-darwin platform.

Reply:
I'm not super sure about that, but please open a PR if necessary.